traits/trait_converters.py

```diff
@@ -42,6 +42,33 @@ def trait_cast(obj):
     return None
 
 
+def as_ctrait(obj):
+    """ Convert to CTrait if the object knows how, else raise TraitError.
+
+    Parameters
+    ----------
+    obj
+        An object that supports conversion to `CTrait`. Refer
+        documentation for when the object supports this conversion.
+
+    Returns
+    -------
+    ctrait.CTrait
+
```

Comment:
Thanks for the updates! We still need a short description here (else the generated documentation just says "Returns: ", with nothing following it).

Reply:
done in a504d64.